internal/exporter/httpsinkexporter/config_test.go

```diff
@@ -21,29 +21,30 @@ import (
 	"github.com/stretchr/testify/assert"
 	"github.com/stretchr/testify/require"
 	"go.opentelemetry.io/collector/component"
-	"go.opentelemetry.io/collector/component/componenttest"
-	"go.opentelemetry.io/collector/config"
-	"go.opentelemetry.io/collector/otelcol/otelcoltest"
+	"go.opentelemetry.io/collector/confmap/confmaptest"
 )
 
 func TestLoadConfig(t *testing.T) {
-	factories, err := componenttest.NopFactories()
-	assert.NoError(t, err)
 
-	factory := NewFactory()
-	factories.Exporters[typeStr] = factory
-	cfg, err := otelcoltest.LoadConfigAndValidate(path.Join(".", "testdata", "config.yaml"), factories)
+	configs, err := confmaptest.LoadConf(path.Join(".", "testdata", "config.yaml"))
```

Comment:
Is removing `LoadConfigAndValidate` dependency update related or are you wanting to minimize the config validation to the components under test?

Reply:
it's a double whammy. Some of the calls in this code are now deprecated, and therefore make lint fail. The way this test works is also antiquated per otel contrib guidelines, where they use this new approach instead.